src/backend/joanie/core/models/events.py

```diff
@@ -0,0 +1,47 @@
+"""
+Declare and configure the models for Joanie's events
+"""
+import logging
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from joanie.core import enums
+from joanie.core.models.base import BaseModel
+
+logger = logging.getLogger(__name__)
+
+
+class Event(BaseModel):
+    """
+    Event represents a message to a user
+    """
+
+    user = models.ForeignKey(
+        to="core.User",
+        verbose_name=_("user"),
+        related_name="events",
+        on_delete=models.CASCADE,
+    )
+    message = models.TextField(_("message"))
+    level = models.CharField(
+        _("level"),
+        max_length=20,
+        choices=enums.EVENT_LEVEL_CHOICES,
+        default=enums.EVENT_INFO,
+    )
+    read = models.BooleanField(_("read"), default=False)
```

Comment:
By convention, we prefix boolean flag with a verb. e.g: `is_main`

Reply:
Initially, this was a Notification model, so `is_read` had sense.
Now, I think we should remove it.